pkg/autoscaler/statserver/server.go

```diff
@@ -41,17 +53,19 @@ type Server struct {
 	stopCh      chan struct{}
 	statsCh     chan<- metrics.StatMessage
 	openClients sync.WaitGroup
+	ownership   bucket.Ownership
```

Comment:
Could this be just a `func(bktName string) bool` and avoid the entire interface?

Reply:
Done